### ISSUE TITLE
Typo: update version number

### ISSUE THREAD

**Issue by rvanlaak** [Zlatkovsky/BuildingOfficeAddins#59]
Version `1.4` is mentioned in the first sentence, while `1.6` should get mentioned.

![image](https://user-images.githubusercontent.com/2707563/50164586-67928c80-02e3-11e9-8f53-63d54a7291c3.png)

**Comment by Zlatkovsky** [comment]
Good catch.  I'll update it next time that I update the book (which at that point would be 1.7... :-) )
